lib/jsonWebToken.js

```diff
@@ -178,17 +179,17 @@ exports.verify = function(jwtString, PEMKey, options, callback) {
   if (typeof payload.exp !== 'number')
     return done(new Error('invalid exp value in payload'));
   if (!options.ignoreExpiration) {
-    if (Math.floor(Date.now() / 1000) >= payload.exp) {
+    if (Math.floor(Date.now() / 1000) >= payload.exp + constants.CLOCK_SKEW) {
```

Comment:
The Date.now() method returns the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.